templates/resources.md.tmpl

```diff
@@ -1,6 +1,10 @@
 ---
 page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+{{ if gt (len (split .Description "[DEPRECATED]")) 1 -}}
```

Comment:
See https://developer.hashicorp.com/terraform/registry/providers/docs#subcategories for details about subcategory, and I got the logic from https://github.com/hashicorp/terraform-plugin-docs/issues/156#issuecomment-1600427216